openedx/core/djangoapps/credit/partition_schemes.py

```diff
@@ -0,0 +1,98 @@
+"""
+Provides partition support to the user service.
+"""
+import logging
+from course_modes.models import CourseMode
+from student.models import CourseEnrollment
+from verify_student.models import SkippedReverification, VerificationStatus
+
+log = logging.getLogger(__name__)
+
+
+class VerificationPartitionScheme(object):
+    """
+    This scheme randomly assigns users into the partition's groups.
+    """
+    NON_VERIFIED_GROUP = 'non_verified_group'
```

Comment:
For consistency with [ICRV access control architecture doc|https://openedx.atlassian.net/wiki/display/AC/In-Course+Reverification+Access+Control] remove `_group` at the end of both placeholders and values:
```python
NON_VERIFIED = 'non_verified'
VERIFIED_ALLOW = 'verified_allow'
VERIFIED_DENY = 'verified_deny'
```